clients/js/genjs.go

```diff
@@ -1381,7 +1387,13 @@ declare namespace {{.ServiceName}} {
     {{range .ErrorTypes}}
     {{.}}
     {{end}}
-  }
+	}
+
+	namespace Models {
```

Comment:
nit: can you change these tabs to spaces so that we avoid mixing tabs/spaces in the outputed TypeScript? I don't think you need to bother with alignment otherwise since it's more trouble than it's worth.